docs/apis/USBMassStoragePlugin.md

```diff
@@ -254,11 +250,11 @@ Triggered after the device partitions are mounted.
 | Name | Type | Description |
 | :-------- | :-------- | :-------- |
 | params | object |  |
-| params.deviceList | array | mount info list of a USB storage device will be returned |
-| params.deviceList[#] | array |  |
-| partitionName | string | The name of the partition being mounted |
-| mountFlags | string | Flags indicating how the partition is mounted |
-| mountPath | string | The mount point path in the file system |
+| params.mountPoints | array | mount info list of a USB storage device will be returned |
```

Comment:
This is not matching the spec. The spec says we also carry device info. Why was it omitted here?